external/storm-kafka/src/jvm/storm/kafka/SpoutConfig.java

```diff
@@ -27,8 +27,7 @@
     public String zkRoot = null;
     public String id = null;
 
-    // if set to true, spout will set Kafka topic as the emitted Stream ID
-    public boolean topicAsStreamId = false;
+    public String outputStreamId;
```

Comment:
Minor, but we can note  here that if outputStreamId not set, spout will set Kafka topic as the emitted Stream ID